docker-compose.yml

```diff
@@ -245,6 +246,49 @@ services:
           - nginx
         ipv4_address: 10.0.0.3
 
+  registry:
+    image: registry:2
+    container_name: registry
+    environment:
+      - VIRTUAL_HOST=registry.localhost.pwn.college
+      - VIRTUAL_PORT=5000
+      - HTTPS_METHOD=noredirect
+      - REGISTRY_AUTH=token
+      - REGISTRY_AUTH_TOKEN_REALM=https://registry.pwn.college/auth/token                     #https://registry.localhost.pwn.college/auth/token
```

Comment:
We should set this up such that these environment variables are configured via .env with `docker compose` to allow easy dev & prod standup.